### ISSUE TITLE
Status: Fail

### ISSUE THREAD

**Issue by cunning-folk** [CunningFolkDev/elevate360_website#198]

**Comment by jeremylenz** [comment]
@cunning-folk The problem was in the new page for James Benton, Ph.D.   There were a few required fields missing.

I've got it working again locally, but Siteleaf seems to be having issues.  I'll keep you updated.

**Comment by cunning-folk** [comment]
Ok awesome

I think at some point those fields need to be exposed at all times so the
client knows.

Is this possible?

**Comment by jeremylenz** [comment]
@cunning-folk Yeah it's the same problem they had last month, just this time it's with a different collection.  I'll add the default fields tonight, and while I'm there I'll also add them for all the other collections that need them.

**Comment by cunning-folk** [comment]
Ok.

If this intermittent work is overwhelming please let me know.

I realize non of this stuff will get done instantly

**Comment by jeremylenz** [comment]
@cunning-folk Closing this issue as Siteleaf support provided a workaround and the site is publishing again.  Let me know if any issues continue.

**Comment by cunning-folk** [comment]
@jeremylenz hi there.

Client reporting that they are having trouble publishing to the 'Team' page.

I took a peek in Siteleaf and see this error

`Error installing Gemfile (RuntimeError)`

I do see that the team member is now published to the live site tho.

https://wholeview.co/about-us/our-staff/

Not sure whats going on. I forwarded an email from the client. Let me know how to assist.

Best

Peter
